ncm-spma/src/main/pan/components/spma/functions-ips.pan

```diff
@@ -0,0 +1,553 @@
+# ${license-info}                                                                                                                                                                                                                            
```

Comment:
WRT +# ${license-info} - yes, it came originally from the old 'functions' template.